ChangeLog.md

```diff
@@ -4,6 +4,14 @@ This project's release branch is `master`. This log is written from the perspect
 
 ## 2019-06-14 - Unreleased
 
+* Remove the "HasView" and "HasRequest" classes, and the general concept of having a type level "app" identifier. Instead, everything is explicitly parametrised on query and request types directly, and the query type is no longer *required* to be a Functor/Align/etc. so that Vessel becomes an option for defining queries and views.
```

Comment:
parameterized (note this is more than just American vs English spelling)

Reply:
I... I think they're both right?

Reply:
Hm, you're right. It's an alternative spelling that seems to be less popular or something... TIL